app/ptxboa_functions.py

```diff
@@ -925,8 +938,65 @@ def content_input_data(api: PtxboaAPI, settings: dict) -> None:
         fig = px.box(df)
         st.plotly_chart(fig, use_container_width=True)
 
+    st.divider()
+    st.subheader("Data that is identical for all regions:")
+
+    input_data_global = input_data.loc[input_data["source_region_code"] == ""]
+
+    # filter processes:
+    # TODO I would like to filter by process type, where do i get this inforation from?
+    list_processes_all = input_data_global["process_code"].unique().tolist()
+
+    list_processes_transport = [
+        element
+        for element in list_processes_all
+        if ("ship" in element or "pipeline" in element)
+    ]
+
+    list_processes_not_transport = [
+        element
+        for element in list_processes_all
+        if element not in list_processes_transport
+    ]
```

Comment:
you can use the get_dimension method from the api, I updated the documentation of it.

```suggestion
    # filter processes:
    processes = api.get_dimension("process")

    list_processes_transport = processes.loc[
        processes["is_transport"], "process_name"
    ].to_list()

    list_processes_not_transport = processes.loc[
        ~processes["is_transport"], "process_name"
    ].to_list()
```
please note that the result of this suggestion will be not completely identical to your old solution: 

difference in process_transport sets: 
`{'LOHC reconversion', 'Autothermal-Reactor (Blue Hydrogen)', 'Methane Regasification', 'Autothermal-Reactor (Blue Hydrogen) incl. Regasification', 'Ammonia reconversion', 'Hydrogen storage', 'electricity transmission', 'electricity storage', 'Green iron ship', 'Hydrogen Regasification', 'Methane compression', 'LOHC conversion', 'Hydrogen Liquification', 'Hydrogen compression', 'Methane Liquefaction'}`

difference in process_not_transport sets:
`{'', 'Autothermal-Reactor (Blue Hydrogen) incl. Regasification', 'Methane compression', 'Methanol reconversion', 'Methane Regasification', 'Methane Liquefaction', 'electricity transmission', 'Hydrogen compression', 'LOHC reconversion', 'Hydrogen Liquification', 'Autothermal-Reactor (Blue Hydrogen)', 'Hydrogen Regasification', 'PV tracking', 'Ammonia reconversion', 'LOHC conversion'}`

Reply:
`api. get_dimension()` will also provide units for you